airbyte-oauth/src/main/java/io/airbyte/oauth/flows/MicrosoftBingAdsOAuthFlow.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.oauth.flows;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.oauth.BaseOAuth2Flow;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Supplier;
+import org.apache.http.client.utils.URIBuilder;
+
+public class MicrosoftBingAdsOAuthFlow extends BaseOAuth2Flow {
```

Comment:
Nice!

I think that
1) This new class needs to be registered in the factory - https://github.com/airbytehq/airbyte/blob/master/airbyte-oauth/src/main/java/io/airbyte/oauth/OAuthImplementationFactory.java#L22
2) Please add a test like https://github.com/airbytehq/airbyte/blob/master/airbyte-oauth/src/test/java/io/airbyte/oauth/flows/DriftOAuthFlowTest.java

Reply:
1. The factory has it registered.
2. Done.